frontend/js/pong/createTournament.js

```diff
@@ -1,21 +1,76 @@
 export async function sendTournamentForm(form) {
-    const formData = new FormData(form);
-    console.log('Envoi du formulaire :', formData);
+    console.log("sendTournamentForm")
+    createWaitingScreenTournament();
+    // const formData = new FormData(form);
```

Comment:
Maybe keep the commented/non-functional code local or on your branch until it's fixed. We would not want this on Master, or I'll call you Axel from now on